Core JS/index.js

```diff
@@ -0,0 +1,12 @@
+var totalFormSubmitted = 0;
+var submittedData = [];
```

Comment:
Could be const, since array is created once and we push to same memory reference, hence const can be used